### PR TITLE
production sync

### PR DESCRIPTION
**Pull request by tonybaltovski**

### CODE REVIEW

File: docs_versioned_docs/version-ros2humble/robots/accessories/manipulators/kinova_gen3_lite.mdx [hunk 1]
```diff
@@ -210,78 +154,6 @@ If you cannot ping the computer or arm, try rebooting and replugging the Kinoca
 
 :::
 
-### Launching the ROS Driver {#kinova_gen3_lite_launch}
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros2humble/robots/accessories/manipulators/kinova_gen3_lite.mdx:213]: There is a reference to #kinova_gen3_lite_launch that now no longer resolves correctly. The reference should be fixed or removed.

File: docs_versioned_docs/version-ros2humble/ros/ros.mdx [hunk 2]
```diff
@@ -70,3 +71,23 @@ Debian drivers are automatically installed on robots as part of the
...
+### [0.3.0] 2024-09-10
+:::warning
+The `StopStatus` message in `clearpath_msgs` has an additional field in the `0.3.0` release. This message type is used by the MicroROS enabled, robot MCU; therefore, the firmware must be updated on all robot platforms, except the A200. See, the [robot installation section](./installation/robot.mdx#firmware-update) for information on updating the firmware.
+:::
+#### New Features
+ - DD100 and DD150 Platform and [Attachments](./config/yaml/platform/attachments/dx1X0.mdx)
+ - R100 Platform and [Attachments](./config/yaml/platform/attachments/r100.mdx)
+ - [Manipulator Generators](./config/generators.mdx#manipulation-generation)
+ - MoveIt! Configuration Generation
+ - [Kinova Gen3 Lite, 6Dof, and 7Dof Arms](./config/yaml/manipulators.mdx#kinova-arms)
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros2humble/ros/ros.mdx:86]: The #kinova-arms anchor doesn't exist yet and should be added in manipulators.mdx.

File: docs_versioned_docs/version-ros2humble/ros/ros.mdx [hunk 3]
```diff
@@ -70,3 +71,23 @@ Debian drivers are automatically installed on robots as part of the
...
+ - [Robotiq 2F 85 and 140 Grippers](./config/yaml/manipulators.mdx#robotiq-grippers)
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros2humble/ros/ros.mdx:87]: The #robotiq-grippers anchor doesn't exist yet and should be added in manipulators.mdx.

**Review by jhiggins-cpr — CHANGES_REQUESTED** [review]
There are a few broken anchors that would be good to fix before merging.

**Comment by tonybaltovski** [comment]
Updated with fixed anchors.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**